BookPlayer/Coordinators/PresentationFlow/BPPushPresentationFlow.swift

```diff
@@ -9,15 +9,15 @@
 import UIKit
 
 /// Handle the push presentation requirements for Coordinators
-public struct BPPushPresentationFlow: BPCoordinatorPresentationFlow {
+struct BPPushPresentationFlow: BPCoordinatorPresentationFlow {
   /// Navigation used for the flow of the coordinator
-  public unowned let navigationController: UINavigationController
+  let navigationController: UINavigationController
   /// First controller on the ``navigationController`` stack before starting the presentation
-  unowned let initiatingController: UIViewController?
```

Comment:
> ... Unlike a weak reference, however, an unowned reference is used when the other instance has the same lifetime or a longer lifetime...

> ### Important
> Use an unowned reference only when you are sure that the reference always refers to an instance that hasn’t been deallocated.
> If you try to access the value of an unowned reference after that instance has been deallocated, you’ll get a runtime error.

[The Swift Programming Language (5.9)](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/automaticreferencecounting#Unowned-References)

---

In the Automatic Reference Counting (ARC) system, `unowned` references are used to address the Strong Reference Cycle problem between classes. However, using `unowned` references comes with a risk of runtime error. This problem occurs when a struct outlives the class it refers to.

In case you are worried that the struct will live too long and keep the class alive with it, I can understand and follow your decision. 😁

Reply:
Yeah the runtime error is the intent, so there's no way of 'holding it wrong' and cause a retain cycle unintentionally